app/lib/three_scale/spam_protection/protector.rb

```diff
@@ -89,6 +89,33 @@ def to_str
       end
 
       alias to_s to_str
+
+      private
+
+      def request_session
+        return {} if template.controller.blank?
+
+        template.controller.request.session
+      end
+
+      def spam_post_request?
+        !http_method.get? && considered_as_spam?
+      end
+
+      def spam_get_request?
+        http_method.get? && was_considered_as_spam?
+      end
+
+      def was_considered_as_spam?
```

Comment:
```suggestion
      def has_been_considered_as_spam?
```

The condition seems to suggest it has been and it is _still_ considered a possible spam.